package.json

```
@@ -22,26 +22,22 @@
     "release:stable": "dripip stable",
     "release:pr": "dripip pr",
     "clean": "rm -rf dist",
-    "build": "ts-node scripts/ensure-prisma2-version && yarn -s clean && yarn -s build:module-facades && tsc && ts-node scripts/post-build",
+    "build": "yarn -s clean && yarn -s build:module-facades && tsc && ts-node scripts/post-build",
```

```suggestion
    "build": "yarn clean && yarn build:module-facades && tsc && ts-node scripts/post-build",
```

Recently learned yarn is smart enough to do this automatically within scripts

---

That's cool, updated 👍